lessons/LargeDatasetPython/lesson.md

```diff
@@ -0,0 +1,22 @@
+---
+layout: page
+title:
+visible: true
+---
+<!-- change visible to true if you want it on the site -->
+
+# Large Datasets in Python (Aug 25, 2016)
+
+ - **Authors**: Wei Zhan
+ - **Research field**: Immunology
+ - **Lesson topic**: Processing gDNA chip results and single cell PCR results; finding shared motifs.
+ - **Lesson content URL**: <https://github.com/gattacagithub/studyGroup/tree/gh-pages/lessons/LargeDatasetPython>
```

Comment:
See comment below. Change link to: `https://github.com/uoftcoders/studyGroup/tree/gh-pages/lessons/LargeDatasetPython`

Thanks @gattacagithub!